dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/security/oidc/OidcDisabledCondition.java

```diff
@@ -41,6 +41,10 @@
     @Override
     public boolean matches( ConditionContext context, AnnotatedTypeMetadata metadata )
     {
+        if ( isTestRun( context ) )
```

Comment:
@netroms Do you see any problem with doing this? Should only be in effect during tests.

Reply:
I'm not quite sure this is necessary, this oidc feature will only affect a very specific code path when doing the oidc authentication.

Reply:
One of the integration test file failed to load context because it tries to instantiate "DefaultDhisConfigurationProvider" from here, even though "TestDhisConfigurationProvider" should have been loaded instead. This was the simplest way to stop spring from loading DefaultDhisConfigurationProvider. So if it does not do any harm, then let it stay here to ensure the test passes always :)

Reply:
Ah, I can see that now. Well then it makes a lot of sense :) Thanks Ameen!